scripts/views/mapboxView.js

```diff
@@ -373,17 +428,21 @@
   mapboxView.toggleBorders = function () {
     $('.border-toggle').removeClass('active');
     $(this).addClass('active');
+    // turn off any selected layers and remove listeners 
+    map.setLayoutProperty('states-house-districts-selected', 'visibility', 'none');
+    map.setLayoutProperty('states-senate-districts-selected', 'visibility', 'none');
```

Comment:
@Alan-mag just add a setLayoutProperty to turn of the federal selected districts here.

Reply:
alright, great - that is in now w/the lastest commit